tests/untestable/gdb/prettyPrinterTestInvoke

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
```

Comment:
The filename should be ``gdb_pretty_printer_test.sh``.

Reply:
Well I checked it, and a very common convention for executable shell scripts on Linux is to leave away the file extension. The file type is then specified with the ``#!`` line at the beginning of the file. Editors are pretty good at detecting the file format this way. Shell libraries on the other hand that do not have the executable bit set should have the extension `.sh` and no ``#!`` at the beginning of the script. This is an executable and therefore I would prefer it without file extension. But I can take over snake case if it makes you happy.

Reply:
I don't know why you think you need to lecture me about Unix's barbaric conventions. In Nim's repo filenames have an extension, just like in Nim code variables have a type.